apps/client/src/components/user-toasts/user-toasts.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Dialog,
+  DialogTitle,
+  createTheme,
+  DialogContent,
+  ThemeProvider,
+} from '@mui/material';
+import { useEffect, useState } from 'react';
+import {
+  useLoginMutation,
+  useSignUpMutation,
+  useGetAllToastsByUserQuery,
+} from '../../store';
+import { Card } from '../card';
+import { ToastsCard } from '../toasts-card';
+
+type Props = {
+  option: string;
+};
+
+export const DisplayToasts: React.FC<Props> = ({ option }) => {
+  const [dialogOpen, setDialogOpen] = useState<boolean>(false);
```

Comment:
This `UseState` and `UseEffect` are Unecceserry. just use that option condition inside the `open` prop